ballerina-shell/modules/shell-cli/src/main/java/io/ballerina/shell/cli/jline/JlineSimpleCompleter.java

```diff
@@ -46,17 +46,19 @@
  * @since 2.0.0
  */
 public class JlineSimpleCompleter implements Completer {
-    private final StringsCompleter topicsCompleter;
     private final StringsCompleter topicsOptionCompleter;
     private final StringsCompleter commandsCompleter;
     private final StringsCompleter keywordsCompleter;
     private final Completers.FileNameCompleter fileNameCompleter;
+    private final StringsCompleter topicsCompleter;
 
     public JlineSimpleCompleter() {
-        List<String> topicsKeywords = FileUtils.readKeywords(PropertiesLoader.getProperty(TOPICS_FILE));
+
```

Comment:
Please pay attention to formatting.
```suggestion
```

Reply:
Updated in 6d6f51ebb95023c10e576e195cc93ed3ad6e5ecf